src/api/app/models/staging/workflow.rb

```diff
@@ -60,4 +61,21 @@ def create_staging_projects
       staging_project.store
     end
   end
+
+  def update_staging_projects_managers_group
+    return unless changes[:managers_group_id]
+
+    old_managers_group = Group.find(changes[:managers_group_id].first)
+    new_managers_group = managers_group
+
+    staging_projects.each do |staging_project|
```

Comment:
This could also happen in the StagingProject callback, no? So you just have to iterate over the projects and store?

Reply:
No, the callback in `StagingProject` is only a  `before_create`, on update it won't be called

Reply:
But why? I mean it could be also a ``before_save`` and we always update when it changed?